Formula/vdirsyncer.rb

```diff
@@ -1,8 +1,9 @@
+# coding: utf-8
```

Comment:
Sorry, emacs was quite insistent to add this line automagically. Should be fine now and I managed to convince my favourite editor to abide my commands in future ;)